addons/product/security/ir.model.access.csv

```diff
@@ -1,7 +1,6 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 access_product_category_user,product.category.user,model_product_category,base.group_user,1,0,0,0

```

Comment:
don't you need to remove this one too? Same for `access_product_category_manager`?

Reply:
Also, looks like there's a reference to product category in `website_sale`:

https://github.com/odoo/odoo/blob/cc919f3b59ef18bd95edf42eb354a627c4041b68/addons/website_sale/models/product_template.py#L913

I guess you'll have to remove that + where it's used too? Err, looks like the test that uses it was added recently :crying_cat_face: https://github.com/odoo/odoo/blob/cc919f3b59ef18bd95edf42eb354a627c4041b68/addons/website_sale/tests/test_website_sale_product_template.py#L46

Reply:
what's the issue? Are you modifying something on product categories too ? Because this new test is only about the product category, not the uom one :thinking:

Reply:
Ah duh, I misread it. Indeed, @mohammadhossam you accidentally removed the a security settings for `model_product_category` in PoS's access.csv (and that happened to be the one I copied :see_no_evil:)